<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WM_XDYNAMICS_CAR</class>
 <widget class="QMainWindow" name="WM_XDYNAMICS_CAR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xdynamics car</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="GraphicArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>466</width>
         <height>661</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="RightWidgetLayout">
      <item>
       <widget class="QGroupBox" name="GB_Apply_Suspension">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>385</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>385</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Apply Suspension</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>20</y>
           <width>361</width>
           <height>18</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="CheckBoxLayout">
          <item>
           <widget class="QRadioButton" name="RB_FullCar">
            <property name="text">
             <string>Full Car</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_QuarterCar">
            <property name="text">
             <string>Quarter Car</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_TestBed">
            <property name="text">
             <string>Test bed</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="ModelingTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>385</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>385</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="CarModeling_Tab">
         <attribute name="title">
          <string>Car Modeling</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="RoadModeling_Tab">
         <attribute name="title">
          <string>Road Modeling</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="Simulation_Tab">
         <attribute name="title">
          <string>Simulation</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>385</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>385</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xdynamics_car.qrc"/>
 </resources>
 <connections/>
</ui>
